bootstrap/config/manager/kustomization.yaml

```diff
@@ -5,4 +5,4 @@ kind: Kustomization
 images:
 - name: controller
   newName: ghcr.io/k3s-io/cluster-api-k3s/bootstrap-controller
-  newTag: v0.1.7
+  newTag: dev
```

Comment:
This setting seems not used by any other places. Well, if e2e relies on this setting to be 'dev', then you'd better add a comment here, telling other guys not touching it :)